dnf-behave-tests/features/makecache.feature

```diff
@@ -0,0 +1,13 @@
+Feature: makecache command 
+
+
+@xfail
+@fixture.httpd
+@1745170
```

Comment:
Missing a `bz` in the tag... Might wanna reference the bug in the commit message as well?

Also, why mark it `@xfail`, if there's a fix that will be merged along with this? For these, I add the `@not.with_os=rhel__eq__8` tag because the fix won't be on RHEL for some time.

Reply:
oh, yes thank you, I will update that.

About the @xfail I use it because I thought if I just put rhel tag and then the bug fix is released first in for example F30 it will be failing in F31, F32.. until its released there.
But I am not sure about the workflow here. Or are the tests run always on upstream?